### PR TITLE
Cherrypick bugfix by tareq for 1.5

### PR DESCRIPTION
**Pull request by tareq89**
1. Pagination issue in team when location is changed [#6481](https://github.com/opencrvs/opencrvs-core/issues/6481)
2. Search Button Incorrectly Enabled with Less Than Two Parameters in Advanced search (after editing search parameters [#7107](https://github.com/opencrvs/opencrvs-core/issues/7107)
3. Translations in countries are not reflected in the certificate for place of birth [#6989](https://github.com/opencrvs/opencrvs-core/issues/6989)
4. Mark as duplicate button should be disabled when reason is not provided [#7079](https://github.com/opencrvs/opencrvs-core/issues/7079)
5. In update modal, multiple item names are showing wrong for marriage declaration [#7046](https://github.com/opencrvs/opencrvs-core/issues/7046)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -62,13 +62,15 @@
...
 - Fix print record page for an unsaved declaration [#6893](https://github.com/opencrvs/opencrvs-core/issues/6893)
 - Fix Reset pagination to default page (1) when location changes in UserList [#6481](https://github.com/opencrvs/opencrvs-core/issues/6481)
 - Fix Disabling 'Mark as duplicate' button when duplicate reason is empty too [#7083](https://github.com/opencrvs/opencrvs-core/pull/7083)
 - Fix certificate overflowing in preview certificate view [#7157](https://github.com/opencrvs/opencrvs-core/pull/7157)
 - Fix records going completely missing when an unexpected error happens in the backend [#7021](https://github.com/opencrvs/opencrvs-core/pull/7021)
+- Fix "Print and issue to groom|bride" is added to a different variable [#7046](https://github.com/opencrvs/opencrvs-core/issues/7046)
+- Fix removed duplicateTrackingId check in createDuplicateTask method [#7024](https://github.com/opencrvs/opencrvs-core/issues/7024)
+- Fix address property handling and corrected country data transformation logic [#6989](https://github.com/opencrvs/opencrvs-core/issues/6989)
+- Fix client modal glitches on integrations page [#7002] (https://github.com/opencrvs/opencrvs-core/issues/7002)
```
Comment (Zangetsu101) [inline CHANGELOG.md:73]: This looks to be a duplicate

File: packages/client/src/i18n/messages/form.ts [hunk 2]
```diff
@@ -222,6 +222,16 @@ export const formMessageDescriptors = {
...
+  certifyRecordToGroom: {
+    defaultMessage: 'Print and issue to groom',
+    description: 'Label for groom select option to certify record',
+    id: 'form.field.label.app.certifyRecordTo.groom'
+  },
+  certifyRecordToBride: {
+    defaultMessage: 'Print and issue to bride',
+    description: 'Label for bride select option to certify record',
+    id: 'form.field.label.app.certifyRecordTo.bride'
+  },
```
Comment (Zangetsu101) [inline packages/client/src/i18n/messages/form.ts:234]: Has the corresponding countryconfig changes been cherry-picked?

File: CHANGELOG.md (hunk 1 above)
Comment (tareq89) [reply CHANGELOG.md:73]: is it? I do not see any repetitions of these lines in the file.

File: packages/client/src/i18n/messages/form.ts (hunk 2 above)
Comment (tareq89) [reply packages/client/src/i18n/messages/form.ts:234]: https://github.com/opencrvs/opencrvs-countryconfig/blob/release-v1.5.0/src/translations/client.csv?plain=1#L914-L918

Yes, its has been.

File: packages/client/src/views/RegisterForm/duplicate/DuplicateForm.test.tsx [hunk 3]
```diff
@@ -107,13 +107,19 @@ describe('when in device of large viewport', () => {
             .props().disabled
         ).toBeTruthy()
       })
-      it('enable the duplicate button on modal when select duplicate Id', async () => {
+      it('enable the duplicate button on modal when select duplicate Id and describe-reason has valu', async () => {
```
Comment (Zangetsu101) [inline packages/client/src/views/RegisterForm/duplicate/DuplicateForm.test.tsx:110]:
```suggestion
      it('enable the duplicate button on modal when select duplicate Id and describe-reason has value', async () => {
```

**Pull request merged**